src/main/resources/spring/camel-context.xml

```diff
@@ -542,19 +532,8 @@
             </aggregate>
         </route>
 
-        <route id="call-kie-extract-reports" trace="true">
-            <from uri="jms:queue:inputDataModel" />
-            <log message="Message received" />
-            <to uri="log:INFO?showBody=true&amp;showHeaders=true" />
-            <bean method="createMigrationAnalyticsCommand(${body})" ref="decisionServerHelper"/>
-            <to id="decisionserver" uri="direct:decisionserver"/>
-            <bean id="route-extract-reports" method="extractReports" ref="decisionServerHelper"/>
-            <log id="route-log-totalPrice" message="totalPrice = ${body.totalPrice}"/>
-            <to uri="jpa:org.jboss.xavier.integrations.migrationanalytics.output.ReportDataModel" />
-        </route>
-
         <route id="route-ma">
-            <from uri="jms:queue:uploadFormInputDataModel" />
+            <from uri="seda:uploadFormInputDataModel?concurrentConsumers={{thread.concurrentConsumers:5}}"/>
```

Comment:
@jonathanvila I think it should be possible to use `direct` instead of `seda`, is there a special reason to use `seda` here?... My point is that we don't need to request an additional Thread to process this piece of code and we can let the original Thread to execute it without blocking and additional Thread. Maybe I'm missing something, what do you think?

Reply:
Here I put the Calls diagram of xavier-integration : https://drive.google.com/file/d/1jBLCldaxChdkcGgXgwtQJFFlLltyfaVb/view?usp=sharing
In my opinion having an asynchronous call to delegate the flow is always beneficial if it's easy to handle.